x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -9,10 +9,15 @@ export function isLegacyEvent(event: ResolverEvent): event is LegacyEndpointEven
   return (event as LegacyEndpointEvent).endgame !== undefined;
 }
 
-export function isProcessStart(event: ResolverEvent): boolean {
+export function isStart(event: ResolverEvent): boolean {
   if (isLegacyEvent(event)) {
     return event.event?.type === 'process_start' || event.event?.action === 'fork_event';
   }
+
+  if (Array.isArray(event.event.type)) {
+    return event.event.type.includes('start');
```

Comment:
Not necessary for this PR if you don't want to wait for CI all over again 😅 , but I would put `start`, `process_start`, `for_event`, etc... all in some kind of constants file. It'll help avoid spelling bugs in any future updates / refactors and also give us a central place to see all these terms.

Reply:
no

Reply:
Good point